manifests/params.pp

```diff
@@ -21,7 +21,7 @@
 
   case $facts['os']['family'] {
     'Debian': {
-      if $facts['os']['release']['major'] =~ /^(8|9|10)$/ {
+      if $facts['os']['release']['major'] =~ /^(9|10)$/ {
         $manage_package_repo       = true
         $nodejs_debug_package_name = 'nodejs-dbg'
```

Comment:
If I look at https://packages.debian.org/stretch/nodejs-dbg then it looks like this package is not available on Buster, but [npm](https://packages.debian.org/buster/npm) is.

Reply:
I will try to enable acceptance tests for Debian 10 to fix this now and in the future